python/lsst/daf/butler/datastores/s3Datastore.py

```diff
@@ -113,6 +113,7 @@ def _artifact_exists(self, location: Location) -> bool:
         exists : `bool`
             True if the location can be found, false otherwise.
         """
+        log.debug(f"Checking if file exists: '{location.uri}''")
```

Comment:
As a general comment for debug log messages we tend to use the `%s` deferred formatting approach following the notion that there is no point stringifying `location.uri` if the debug message is not going to be issued. So all these should ideally be changed to:
```python
log.debug("something or other %s", location.uri)
```